src/content/reference/manifest.mdx

```diff
@@ -509,6 +509,41 @@ metadata:
     custom.label/dev: "true"
 ```
 
+#### mode (string, optional)
+
+Defines how `okteto up` will setup the development container and behave within the cluster.
+
+Currently two options are available:
+- `sync`: (default) Sync & run your code in a remote development container.
+- `hybrid`: Run your code on your local computer instead of in a remote development container, while rest of your service components runs in your remote cluster.
+
+Most use-cases should use default `sync` mode, but `hybrid` may be desired to achieve a faster development experience in some scenarios:
+
+|   | `sync` | `hybrid` |
+|---|---|---|
+| Soure code  | Uses Syncthing to sync files bidirectionally with your remote development container | Uses local source code directly, no files syncing needed |
```

Comment:
both sync and hybrid "Uses local source code directly".  I don't think this is a good way to explain the difference. The main difference is that on sync, the code is synchronized and development runs in a container, while on hybrid, the service under development runs in the local computer, and it "connects" with the rest of the application in the cluster, right?

Reply:
yes, that's it. Maybe is a little bit confusing this table yeah. I don't think it add too much value